### DISCUSSION TITLE
How do I raise the temperature cap from 260 to 300

### DISCUSSION THREAD

**Discussion by BobCraig10** [Jyers/Marlin#821] [Q&A] [answered]
Hi, I installed an all metal hotend and need to raise the temperature cap for the hot end to be able to work with materials needing 300°c,  how would I do this? 
Thanks in advance!

**Comment by dkustec** [comment, marked as answer]
You should buy a brand new heater and the thermistor. Stock thermistor can't go above 270°C. 
Then, you should change your config slightly and enable a new thermistor (probably - `13 : 100k Hisens 3950  1% up to 300°C for hotend "Simple ONE " & "Hotend "All In ONE"`). Configure max temp. and do some PID tune.

**Reply by Reggi3** [reply]
You'll want an all-metal hotend/heatbreak too, you do not want to be messing around with 300c and ptfe bowden tubes touching nozzles, teflon off-gassing  is a health hazard.
